newIDE/electron-app/app/LocalFilesystemWatcher.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+const watcher = require('@parcel/watcher');
+
+let subscriptions = {};
+
+const getNewSubscriptionId = () => {
+  const id = Math.round((Math.random() + 1) * 10000).toString();
+  if (subscriptions[id]) return getNewSubscriptionId();
+  return id;
```

Comment:
ubernit: You could also use a monotonically increasing integer. I.e: `let nextSubscriptionId = 1` and do `nextSubscriptionId++`